### ISSUE TITLE
[Forms] Label not colored with red if input contains errors

### ISSUE THREAD

**Issue by n-srg** [Tradeshift/tradeshift-ui#370]
TSUI v 9.1.1

In case input field contains error: it should render <span> label with red color

#### Expected Behavior
![image](https://user-images.githubusercontent.com/22394106/29875332-e7044e7c-8dc3-11e7-9446-901189e1c372.png)

#### Actual Behavior
![image](https://user-images.githubusercontent.com/22394106/29875324-de06e618-8dc3-11e7-8e69-452b64019d79.png)

#### Steps to reproduce
Go to http://ui.tradeshift.com/#components/forms/example.html
And take a look on example related to handling errors

**Comment by wiredearp** [comment]
@bomouridsen Shouldn't perhaps the blue *focus outline* also be red until the field is marked as valid?

**Comment by jerf** [comment]
@wiredearp yes I think it's best that the outline is red in the error state. I need to review the design files that Bo made to answer whether the label should be red as well - I'm not sure that will be necessary when the outline is red.

**Comment by jerf** [comment]
Yes, outline in red but label stays the same in an error state.

**Comment by sampi** [comment]
This is how it currently looks in v10 on the [docs site](http://ui.tradeshift.com/#components/forms/example.html):

unfocused:
![screen shot 2018-02-06 at 16 57 43](https://user-images.githubusercontent.com/524272/35869261-f1acc69c-0b5e-11e8-8422-50267b2a834e.png)

focused:
![screen shot 2018-02-06 at 16 57 53](https://user-images.githubusercontent.com/524272/35869272-f5872b9a-0b5e-11e8-8a56-80412619057d.png)
